main/glossary/README.md

```diff
@@ -424,9 +436,15 @@ can immediately cause the [seat](#seat) to exit, getting back the amount it offe
 
 ## Passable
 
-A *passable* is something that can be marshalled (see the
-[Marshaling section in the JavaScript Distributed Programming Guide](/guides/js-programming/far.md#marshaling-by-copy-or-by-presence))
-and sent to and from remote objects.
+A *passable* is something that can be sent to and from remote objects.
+Passables include pass-by-copy primitive values such as numbers and strings and
+pass-by-reference values such as Remotables and Promises.
+Passables also include [hardened](#harden) acyclic pass-by-copy containers that
+recursively terminate in non-container passables, such as
+[CopyArrays](#copyarray) and [CopyRecords](#copyrecord).
```

Comment:
> ```md
>  that recursively terminate in non-container passables, such as
> [CopyArrays](#copyarray) and [CopyRecords](#copyrecord).
> ```

That makes it sound like CopyArrays and CopyRecords are examples of "non-container passables".